articles/confidential-ledger/manage-aad-token-based-users.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Managed AAD Token Based Users in Azure Confidential Ledger
+description: Learn how to manage AAD token based users in Azure Confidential Ledger
+author: settiy
+ms.author: settiy
+ms.date: 02/09/2023
+ms.service: confidential-ledger
+ms.topic: how-to
+---
+
+# Manage AAD Token Based Users in Azure Confidential Ledger
+
+AAD-based users are identified by their AAD object id.
+
+Users with Administrator privileges can manage users of the Confidential Ledger. Available roles are Reader (read-only), Contributor (read and write), and Administrator (read, write, and manage users).
```

Comment:
```suggestion
Members with Administrator privileges can manage users of the Confidential Ledger. Available roles are Reader (read-only), Contributor (read and write), and Administrator (read, write, and manage users).
```

Reply:
In ACL, Administrator users can manage other users. They don't have to be members